src/components/shared/footer/Footer.jsx

```diff
@@ -44,8 +44,9 @@ const Footer = () => {
               rel="noopener noreferrer"
               className="footer__content__bottom__mail"
             >
-              {footerContent.mail.address}
+            Contact us: {footerContent.mail.address}
```

Comment:
I guess we can remove this as the previous line says `Email Us for more details`